official/vision/data/tfrecord_lib_test.py

```diff
@@ -87,7 +88,90 @@ def test_convert_to_feature_bytes_list(self):
 
     proto = tfrecord_lib.convert_to_feature([b'123', b'456'])
     self.assertSequenceAlmostEqual(proto.bytes_list.value, [b'123', b'456'])
+  
+  def test_obj_annotation_tf_example(self):
```

Comment:
@luotigerlsx 

I have added the test case, can you please check and review the same. Really thanks for reviewing the code.

Reply:
Thanks for adding the test case. It looks good to me. But due to the newly added code, the `lint` test doesn't pass. Please check the log the modify accordingly.

Reply:
I made all the corrections which are required from `lint` of point of view. Thanks